### PR TITLE
Advanced chart

### PR DESCRIPTION
**Pull request by JasperCraeghs**
Make `:chart:` option more advanced:

- Allow a colon-separated `<<attribute>>` parameter.
- Add an alternative to a list of attribute values: a threshold value for grouping defects with the same attribute value if their amount reaches the threshold value.

Fix typos in `:widths:` option in example code snippet.

Fix cut off labels in pie chart image when labels are too long by increasing pie chart width from 8 to 10.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -193,9 +205,19 @@ work for this attribute, e.g. `MISRA`.
 chart
 -----
 
-This optional, second display option will draw a pie chart that visualizes the amount of results for each specified
-classification option. The parameters can be a list of classification options, comma-separated, or even
-plus-sign-separated to combine multiple options into the same slice.
+This optional, second display option will draw a pie chart that visualizes the amount of results for each allowed
+`<<attribute>>` option. Firstly, the attribute can be specified, followed by a colon. The default attribute is
+`classification`. Secondly, you have two optoins. Either you specify a list of attribue values, comma-separated or even
+plus-sign-separated for a merge into the same slice or else you define the minimum number of defects with the same
```
Comment (Letme) [inline README.rst]: Put comma here somewhere. `Minimum number` can also be `minimum threshold`

File: README.rst [hunk 2]
```diff
@@ -193,9 +205,19 @@ work for this attribute, e.g. `MISRA`.
 chart
 -----
 
-This optional, second display option will draw a pie chart that visualizes the amount of results for each specified
-classification option. The parameters can be a list of classification options, comma-separated, or even
-plus-sign-separated to combine multiple options into the same slice.
+This optional, second display option will draw a pie chart that visualizes the amount of results for each allowed
+`<<attribute>>` option. Firstly, the attribute can be specified, followed by a colon. The default attribute is
```
Comment (Letme) [inline README.rst]: after colon add `:`, so that everyone knows its not `column` intentionally.

File: README.rst (hunk 1 above)
Comment (JasperCraeghs) [reply README.rst]: fdae3884851cf7711b99f8e99c825935c7daabed

File: README.rst (hunk 2 above)
Comment (JasperCraeghs) [reply README.rst]: fdae3884851cf7711b99f8e99c825935c7daabed

**Pull request merged**